compute module: Add note about support / diagnostic tips. (#3946)
raspberrypi/documentation: documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

To flash the Compute Module eMMC, you need the following:

* Another computer, referred to in this guide as the *host device*. You can use Linux (we recommend Raspberry Pi OS or Ubuntu), Windows 11, or macOS.
* The Compute Module IO Board xref:compute-module.adoc#io-board-compatibility[that corresponds to your Compute Module model].
* A micro USB cable, or a USB-C cable for Compute Module models since CM5IO.

> NOTE: If the Compute Module is not recognised then please remove any USB hubs between the *host device* and the Compute Module.
> For more diagnostic tips see https://github.com/raspberrypi/usbboot?tab=readme-ov-file#troubleshooting

Reviewer: ```suggestion
TIP: In some cases, USB hubs can prevent the host device from recognising the Compute Module. If your host device does not recognise the Compute Module, try connecting the Compute Module directly to the host device. For more diagnostic tips, see https://github.com/raspberrypi/usbboot?tab=readme-ov-file#troubleshooting[the usbboot troubleshooting guide].
```

Suggestion: slight reword to clarify 'remove' terminology; improve formatting of troubleshooting link; ensure that troubleshooting link is part of the admonition

Next, let's set up software on the host device.

> The recommended host device is a Raspberry Pi 4 or newer running Raspberry Pi OS 64-bit.

Reviewer: ```suggestion
TIP: For a host device, we recommend a Raspberry Pi 4 or newer running 64-bit Raspberry Pi OS.
```

Suggestion: convert this into a TIP admonition, since it isn't essential to the guide flow.

Windows::
+
To set up software on a Windows 11 host device:
+
. Download the https://github.com/raspberrypi/usbboot/raw/master/win32/rpiboot_setup.exe[Windows installer] or https://github.com/raspberrypi/usbboot[build `rpiboot` from source].

> . Double-click on the installer to run it. This installs the drivers and boot tool. Do not close any driver installation windows which appear in the installation process.

Reviewer: ```suggestion
. Double-click on the installer to run it. This installs the drivers and boot tool. Do not close any driver installation windows which appear during the installation process.
```
